tendrl_server.yml

```diff
@@ -2,15 +2,18 @@
 #
 # Install Tendrl server
 #
-- hosts: usm_server 
+- hosts: usm_server
   remote_user: root
   vars:
     install_from: packages
   roles:
     - etcd
     - tendrl-repo
+    - ceph-installer-repo
     - { role: epel, epel_enabled: 1 } # based on https://github.com/Tendrl/tendrl-api/issues/25
     - tendrl-api
+    - tendrl-node-agent
+    - ceph-installer
```

Comment:
In the commit message which add these lines, it would be nice to link to related Tendrl documentation/feature/commit.